### PR TITLE
Handle missing default model

### PR DESCRIPTION
**Pull request by lmdehaas**
Adds MissingModelException for the case when default Spacy model is not available

### CODE REVIEW

File: textpipe/doc.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def _load_spacy_doc(self, lang, model_name=None):
                                             model_name not in self._spacy_nlps[lang]):
             self._set_default_nlp(lang)
         if model_name not in self._spacy_nlps[lang] and model_name is not None:
-            raise TextpipeMissingModelException(f'Custom model {model_name} '
+            raise MissingModelException(f'Custom model {model_name} '
                                                 f'is missing.')
```
Comment (babakx) [inline textpipe/doc.py]: the spacing here needs to be fixed (remove extra spaces).

File: textpipe/doc.py [hunk 2]
```diff
@@ -286,7 +286,10 @@ def _set_default_nlp(self, lang):
         """
         if lang not in self._spacy_nlps:
             self._spacy_nlps[lang] = {}
-        model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
+        try:
+            model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
+        except IOError:
+            raise MissingModelException(f'Default model for language "{lang}" is not available.')
```
Comment (dodijk) [inline textpipe/doc.py]: Could the be another reason for an `IOError` at this point? Not sure. Might be OK for now.

File: textpipe/doc.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 from bs4 import BeautifulSoup
 
 
-class TextpipeMissingModelException(Exception):
+class MissingModelException(Exception):
```
Comment (anneschuth) [inline textpipe/doc.py]: This rename breaks backwards compatibility. Don't think we need it? If we do, we should bump the version to 7.

Comment (lmdehaas) [reply textpipe/doc.py]: Fixed (reverted rename)

File: textpipe/doc.py (hunk 2 above)
Comment (lmdehaas) [reply textpipe/doc.py]: I don't think so: spacy tries to find only this file. I think this should be fine.

**Pull request merged**